.github/workflows/ci.yml

```diff
@@ -46,6 +47,18 @@ jobs:
       - name: Install NPM dependencies
         run: npm install
 
+      - name: Install Spago dependencies
+        run: spago install
```

Comment:
This is fine to add, if you'd like, but it's technically unnecessary because Spago will automatically install packages before building them if you call `spago build`.